sdk/aqualink_sdk/api/aggregate.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from pprint import pprint
+
+default_mapping = {
+    'min_temperature':'min',
+    'precipitation':'sum',
+    'alert_level':'max',
+    'bottom_temperature':'mean'
+}
+
+# TODO - Build on top of TimeSeriesApi ?
+
+def aggregate_data(timeseries_data, aggregate_frequency, aggregate_mapping=default_mapping):
```

Comment:
@laurentS I would love to get your thoughts on this first iteration of aggregator functions